compiler/compiler.go

```diff
@@ -391,8 +402,13 @@ func (c *compileCtx) getType(name string) (protobuf.Type, error) {
 		}
 
 		for t := range container {
-			if t.Name() == name {
-				return t, nil
+			if name == "array" {
+				c.addImportForType("google.protobuf.ListValue")
```

Comment:
I need to read through this deeper, but is this adding `ListValue` as an import for _any_ nested array?

Reply:
This was removed as it is now using static list wrapper types.